enatega-multivendor-app/src/components/CustomizeComponents/HeadingComponent/HeadingComponent.js

```
@@ -8,12 +8,19 @@ import TextDefault from '../../Text/TextDefault/TextDefault'
 import { scale } from '../../../utils/scaling'
+import { calculateDiscountedPrice } from '../../../utils/calculateDiscountedPrice'
 
 function HeadingComponent(props) {
   const { i18n } = useTranslation()
   const configuration = useContext(ConfigurationContext)
   const themeContext = useContext(ThemeContext)
   const currentTheme = { isRTL: i18n.dir() === 'rtl', ...theme[themeContext.ThemeValue] }
+  console.log(props?.price, props?.discountedPrice);
+
+  const withoutDiscountPrice = calculateDiscountedPrice(props?.price, props?.discountedPrice)
+  console.log(withoutDiscountPrice, "withoutDiscountPrice in HeadingComponent");
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)